jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -46,12 +47,25 @@
  */
 public class DockerContextGenerator {
 
+  /**
+   * Builds a list of Dockerfile "EXPOSE" instructions.
+   *
+   * @param exposedPorts the list of ports numbers/ranges to expose
+   * @return a string containing an EXPOSE instruction for each of the entries
+   */
+  @VisibleForTesting
+  static String makeExposeInstructions(List<String> exposedPorts) {
```

Comment:
This could also be moved to `makeDockerfile` and change that to append to a `StringBuilder` as an alternative.